addons/br/nfse/item_test.go

```diff
@@ -0,0 +1,62 @@
+package nfse_test
+
+import (
+	"testing"
+
+	"github.com/invopop/gobl/addons/br/nfse"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestItemValidation(t *testing.T) {
+	tests := []struct {
+		name string
+		item *org.Item
+		err  string
+	}{
+		{
+			name: "valid item",
+			item: &org.Item{
+				Ext: tax.Extensions{
+					nfse.ExtKeyService: "12345678",
+				},
+			},
+		},
+		{
+			name: "missing extensions",
+			item: &org.Item{},
+			err:  "ext: (br-nfse-service: required.)",
+		},
+		{
+			name: "empty extensions",
+			item: &org.Item{
+				Ext: tax.Extensions{},
+			},
+			err: "ext: (br-nfse-service: required.)",
+		},
+		{
+			name: "missing extension",
+			item: &org.Item{
+				Ext: tax.Extensions{
+					"random": "12345678",
+				},
+			},
+			err: "ext: (br-nfse-service: required.).",
+		},
```

Comment:
You could test the nil case here like this:
```suggestion
		},
		{
		  name: "nil",
		  item: nil,
		},
```